components/x-podcast-launchers/readme.md

```diff
@@ -15,21 +15,33 @@ The [`x-engine`][engine] module is used to inject your chosen runtime into the c
 
 [engine]: https://github.com/Financial-Times/x-dash/tree/master/packages/x-engine
 
+## Styling
+
+To get correct styling, Your app should have origami components below.  
+[o-typography](https://registry.origami.ft.com/components/o-typography)  
+[o-buttons](https://registry.origami.ft.com/components/o-buttons) v5  
+[o-forms](https://registry.origami.ft.com/components/o-forms)  v6 (v7 above breaks the styling)
 
 ## Usage
 
 The components provided by this module are all functions that expect a map of [properties](#properties). They can be used with vanilla JavaScript or JSX (If you are not familiar check out [WTF is JSX][jsx-wtf] first). For example if you were writing your application using React you could use the component like this:
 
 ```jsx
 import React from 'react';
-import { PodcastLaunchers } from '@financial-times/x-podcastlaunchers';
+import { PodcastLaunchers } from '@financial-times/x-podcast-launchers';
 
 // A == B == C
 const a = PodcastLaunchers(props);
 const b = <PodcastLaunchers {...props} />;
 const c = React.createElement(PodcastLaunchers, props);
 ```
 
+```scss
+// within your app's sass file
+@import "x-podcast-launchers/dist/PodcastLaunchers";
+```
+:warning: It includes classes to be overwritten o-forms styling. Therefore it needs to be imported after o-forms is imported;
```

Comment:
Could this be worded as something like this?

> Warning: this component depends on styles provided by o-forms, and therefore o-forms needs to be imported before x-podcast-launchers.

Reply:
Or even:

> and therefore o-forms needs to be imported alongside this component.

Reply:
@benbarnett Thank you for your suggestion! I will change the sentences as you suggested.